### ISSUE TITLE
Invisible labels of exported images during dark mode

### ISSUE THREAD

**Issue by Jordatious** [CARTAvis/carta#19]
When exporting images (including plots) during dark mode, the labels in the output figure are invisible, or otherwise difficult to see, since they're white, over a gray background, which seems to be semi-transparent. Using black would be much more useful, as when exporting images during light mode. I can't find a preference to configure the label colours, but perhaps there is, in which case the default should be updated.

**Comment by kswang1029** [comment]
The overlay settings dialogue (accessible via the button at the top) should allow users to change the color of axis labels. 
![541106F5-5E58-480A-BDCD-2B200CF6A090](https://user-images.githubusercontent.com/20819712/79244144-1bd8fe80-7ea9-11ea-8e22-a87e7b617913.jpeg)

**Comment by veggiesaurus** [comment]
> over a gray background, which seems to be semi-transparent

The output figure's background is actually fully transparent, so you can paste it into presentations with an arbitrary background colour.

**Comment by Jordatious** [comment]
Yep, I can change it in the overlay settings, but when I restart my session, it resets. So I was suggesting it would be good to have a default for dark mode that isn't white. When one just opens the png in preview for Mac OS X, and presumably other apps, it's very difficult to see.

**Comment by veggiesaurus** [comment]
@kswang1029 shall we add a setting in preferences for exporting with opaque or transparent backgrounds? For embedding images in presentations, a transparent background is better, but for simply saving images, an opaque background is preferred, as @Jordatious mentioned.
